libs/authorizenet/src/authorize-net.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DaffAuthorizeNetStateModule } from './authorize-net-state.module';
+
+const ACCEPT_LIBRARY = 'https://jstest.authorize.net/v1/Accept.js';
+
+@NgModule({
+  imports: [
+    CommonModule,
+
+    /**
+     * Ngrx/store
+     */
+    DaffAuthorizeNetStateModule
+  ]
+})
+export class DaffAuthorizeNetModule { 
+	constructor() {
+		const acceptJsScript = document.createElement('script');
```

Comment:
This probably shouldn't be in the constructor of the module. We should have a system to determine when the`accept.js` lib is loaded successfully.

Reply:
moved to an action and effect